### PR TITLE
Replace if-else with switch for readability

### PR DESCRIPTION
**Pull request by kerthcet**
Signed-off-by: kerthcet <kerthcet@gmail.com>

#### What type of PR is this?
/kind cleanup

#### Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: pkg/controller/core/workload_controller.go [hunk 1]
```diff
@@ -82,27 +82,31 @@ func (r *WorkloadReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
...
+		if !r.cache.ClusterQueueActive(cqName) {
+			err := workload.UpdateStatusIfChanged(ctx, r.client, &wl, kueue.WorkloadAdmitted, corev1.ConditionFalse,
+				"Inadmissible", fmt.Sprintf("ClusterQueue %s is inactive", cqName))
+			return ctrl.Result{}, client.IgnoreNotFound(err)
+		}
+	case admitted:
 		err := workload.UpdateStatusIfChanged(ctx, r.client, &wl, kueue.WorkloadAdmitted, corev1.ConditionTrue, "", "")
 		return ctrl.Result{}, client.IgnoreNotFound(err)
+	default:
+		return ctrl.Result{}, nil
```
Comment (alculquicondor) [inline pkg/controller/core/workload_controller.go:109]: nit: you could remove these lines

**Review by alculquicondor — COMMENTED** [review]
/approve
/lgtm
/hold
in case you want to do the nit

File: pkg/controller/core/workload_controller.go (hunk 1 above)
Comment (kerthcet) [reply pkg/controller/core/workload_controller.go]: Updated.

**Pull request merged**